ui/packages/shared/profile/src/TopTable/index.tsx

```diff
@@ -142,22 +140,27 @@ export const TopTable = ({
 
   const onRowClick = useCallback(
     (row: TopNode) => {
+      // If there is only one dashboard item, we don't want to select a span
+      if (dashboardItems.length <= 1) {
```

Comment:
Should we also not show the pointer cursor for the table items when this condition matches? 
As that gives a wrong impression that it is clickable but nothing happens on click.

Reply:
good idea! i'll implement that